### DISCUSSION TITLE
【Bing SEO合规性要求】图片的Alt属性、网页H1标题和 Html 大小太长

### DISCUSSION THREAD

**Discussion by qixing-jk** [tangly1024/NotionNext#2120] [Ideas]
**为什么提出这个新的特性改动**
bing seo要求：
- 缺少图像的 Alt 属性
- 缺少 h1 标记
- Html 大小太长

多处Alt属性缺失，经检查有如下部分
当前文章的自定义图标和封面页
相关文章、最新发布模块的封面页

**描述一下你推荐的解决方案**
- 缺少图像的 Alt 属性：为其设置默认值，自动配置为其文章标题即可
- 缺少 h1 标记：将主标题配置为H1标题
- Html 大小太长：应该是包含大量的 CSS 或代码，具体不知如何解决

**描述一下你考虑过的其它替代解决方案**
暂无

**补充说明**
补充与此特性相关的内容

**Comment by wuyuhanzijin** [comment]
是的，必应站长工具给出的严重SEO问题是这些
